### ISSUE TITLE
Call to Tone analyzer with Json fails 

### ISSUE THREAD

**Issue by hande1** [watson-developer-cloud/python-sdk#319]
I'm trying to use tone_analyzer. When I send tweets one-by-one, it works well. If I send a huge json, I'm getting this error. 
I'd appreciate any help.

```python
Traceback (most recent call last):
  File "tone_1201.py", line 33, in <module>
    tone = tone_analyzer.tone(json.load(jsonfile),content_type='application/json')
  File "/usr/local/lib/python2.7/dist-packages/watson_developer_cloud/tone_analyzer_v3.py", line 75, in tone
    json=text, accept_json=True)
  File "/usr/local/lib/python2.7/dist-packages/watson_developer_cloud/watson_developer_cloud_service.py", line 324, in request
    raise WatsonException(error_message)
watson_developer_cloud.watson_developer_cloud_service.WatsonException: Unknown error, Code: 500
```

**Comment by germanattanasio** [comment]
@hande1 which version of the SDK are you using?

**Comment by ehdsouza** [comment]
@hande1 you could also provide a snippet so that we could help you better.

**Comment by hande1** [comment]
The version is : watson-developer-cloud==0.26.1

for fn in os.listdir(directory):
	if fn[len(fn)-4:len(fn)] == 'json':
		OutFile = directory +fn[0:len(fn)-4]+'_ans.json'
		with open(os.path.join(directory, fn), 'r') as jsonfile:
		   tone = tone_analyzer.tone(json.load(jsonfile),content_type='application/json')
		print(json.dumps(tone, indent=1))
		write_json( tone, OutFile)

**Comment by ehdsouza** [comment]
Could you use the latest release and try? v1.0.1 is the latest

**Comment by hande1** [comment]
After writing you, I realized it's not the current version, upgraded, still the same error.

**Comment by Patataman** [comment]
Stupid question: Are you sure you are sending a json file?

More likely problem: Inside the [tone method documentation](https://github.com/watson-developer-cloud/python-sdk/blob/master/watson_developer_cloud/tone_analyzer_v3.py#L128) it says: 

> You can submit no more than 128 KB of total input content and no more than 1000 individual sentences in JSON, plain text, or HTML format

> :param ToneInput tone_input: JSON, plain text, or HTML input that contains the content to be analyzed. For JSON input, provide an object of type `ToneInput`.

Maybe your problem is related with some of those 2 restrictions.

**Comment by hande1** [comment]
yes, I dig the documentation for the file size restriction, couldn't find, but, I was almost sure because the connection is dead after some seconds. With 10 tweets in Json, it's working. I tried the system to work with 20000 tweets :)
So, the aim was doing the sentimental analysis on the tweets that contain particular hashtags. 
I tried the same thing with real time twitter data (no hashtag filter) on IBM cloud and think it's better to use it on IBM cloud with spark and real time tweets.

Thanks to all of you,

Hande
